src/lib/engine.ml

```diff
@@ -37,7 +37,7 @@ include Make_module_error_and_info(struct
 let create configuration =
   Persistent_data.create
     (Configuration.database_parameters configuration)
-    (Configuration.archival_age_threshold configuration)
+    (* (Configuration.archival_age_threshold configuration) *)
```

Comment:
Oops right I left it commented to remind me to https://github.com/hammerlab/ketrew/issues/478